### PR TITLE
Remove test that times out because it legitimately needs longer than …

### PR DESCRIPTION
**Pull request by benjamin-confino**
…the timeout to run

### CODE REVIEW

**Review by Azquelt — DISMISSED** [review]

**Comment by benjamin-confino** [comment]
#build

File: dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/publish/tckRunner/tck/tck-suite-lite.xml [hunk 1]
```diff
@@ -23,11 +23,13 @@
...
                 </methods>
             </class>
+            <!--
+            This was removed because the test could legitimately take longer than the 2second timeout. This bug is fixed in TCK version 3
             <class name="org.eclipse.microprofile.fault.tolerance.tck.CircuitBreakerBulkheadTest">
                 <methods>
                     <include name="testCircuitBreakerAroundBulkheadSync"/>
                 </methods>
-            </class>
+            </class> -->
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/publish/tckRunner/tck/tck-suite-lite.xml]: Actually, I don't think we need to document this.

It's sufficient that we document it being excluded in the full suite XML and here it adds clutter, making it harder to check what is included.

**Review by Azquelt — APPROVED** [review]

**Comment by benjamin-confino** [comment]
The previous build hasn't started yet so it will have the latest change. (Not that we really need a build)

**Comment by benjamin-confino** [comment]
#libby

**Pull request merged**